types/backbone/index.d.ts

```diff
@@ -534,15 +534,15 @@ declare namespace Backbone {
         model?: TModel;
         // TODO: quickfix, this can't be fixed easy. The collection does not need to have the same model as the parent view.
         collection?: Collection<any>; // was: Collection<TModel>;
-        el?: any;
-        events?: EventsHash;
+        el?: HTMLElement | JQuery | string;
         id?: string;
+        attributes?: { [id: string]: any };
         className?: string;
         tagName?: string;
-        attributes?: { [id: string]: any };
+        events?: EventsHash | (() => EventsHash);
     }
 
-    class View<TModel extends Model = Model> extends EventsMixin implements Events {
+    class View<TModel extends Model = Model> extends EventsMixin implements Events, ViewOptions<TModel> {
```

Comment:
`implements ViewOptions<TModel>` doesn't seem right. Sure, the members of `ViewOptions` happen to exist on `View` as well, but I think `implements` is supposed to convey an is-a relation. A `View` is not a special case of `ViewOptions`; you're not supposed to pass a `View` by means of passing options. Also, some `View` subclasses might accept options that don't translate directly to instance properties.

Reply:
I thought that this is correct, since these parameters are directly assigned to the object, https://github.com/jashkenas/backbone/blob/master/backbone.js#L1298 and this can protect us against wrong types in the class itself. Typescript sometimes looks strange, you can extend the interface from the class)
But I do not insist here.

Reply:
Yes, the standard viewoptions are assigned to the instance, but that doesn't mean that `View` *implements* `ViewOptions`. Or that it means anything to state that anything implements `ViewOptions`, for that matter, since there is nothing to implement about `ViewOptions`. It's just a plain carrier of information. `View`s are *not* plain carriers of information.

Compare it to `Events`. There is actually an inheritance-like relationship between `Events` and (the prototype of) `View`. No such thing is going on between `ViewOptions` and `View`; in the actual library, there isn't even a constructor for `ViewOptions`. It's just a list of properties that will be copied if they are present in the *plain object* that is passed as options hash.